app/src/main/kotlin/com/wire/android/ui/home/conversations/ConversationViewModel.kt

```diff
@@ -273,11 +273,6 @@ class ConversationViewModel @Inject constructor(
 
     private suspend fun mapToMessageUI(assetContent: AssetContent, conversationId: ConversationId, messageId: String): MessageContent {
         with(assetContent) {
-            val assetId = remoteData.assetId
-
-            // TODO: To be changed once the error behavior has been defined with product
-            if (assetId.isEmpty()) return TextMessage(messageBody = MessageBody("The asset message could not be downloaded correctly"))
```

Comment:
so downloading assets isn't implemented yet and the same with handling the situation when `assetId` is empty, right? 🤔

Reply:
Exactly. Until we don't implement the downloading behavior, this is not relevant. But yes, once we do, we will need to figure out how to address the situation where the asset ID might be empty in between those 2 messages